README.md

```diff
@@ -14,6 +14,7 @@ quickMARC API provides the following URLs:
 |  Method | URL| Permissions  | Description  | 
 |---|---|---|---|
 | GET | /records-editor/records?instanceId={instanceId}  |records-editor.records.item.get   | Retrieves QuickMarcJson by instance's id  |
+| POST| /records-editor/records|records-editor.records.item.post| Duplicate existing MARK and Instance records|
```

Comment:
"Duplicate" is not the right word here. The endpoint creates new Marc bib record and other related records

Reply:
substituted to a "Create"